9.4/docker-entrypoint.sh

```diff
@@ -9,9 +15,17 @@ if [ "$1" = 'postgres' ]; then
 	
 	if [ -z "$(ls -A "$PGDATA")" ]; then
 		gosu postgres initdb
-		
-		sed -ri "s/^#(listen_addresses\s*=\s*)\S+/\1'*'/" "$PGDATA"/postgresql.conf
-		
+
+        alter_system "listen_addresses" "*"
+
+        conf_prefix="POSTGRES_CONF_"
+        for env_name in $(env | cut -d= -f1 | grep "$conf_prefix.*"); do
```

Comment:
I think this can just be `for env_name in ${!POSTGRES_CONF_*}; do` at the expense of repeating "POSTGRES_CONF_" below.